README.md

```diff
@@ -170,7 +170,7 @@ public void onRationaleDenied(int requestCode) {
 }
 ```
 
-Rationale callbacks don't necessarily imply permission changes. To check for those, see the EasyPermissions.PermissionCallbacks.
+Rationale callbacks don't necessarily imply permission changes. To check for those, see the `EasyPermissions.RationaleCallbacks`.
```

Comment:
Whoops, looks like this should stay `PermissionCallbacks` 😉

Reply:
Smile.. Will create a new PR in a min!

Reply:
Alrighty, go for it!